core/modules/simpletest/tests/file.test

```diff
@@ -364,11 +364,11 @@ class FileValidatorTest extends BackdropWebTestCase {
   function setUp() {
     parent::setUp(array('file'));
 
-    $this->image = new File(array());
+    $this->image = entity_create('file', array());
```

Comment:
Not a big deal, as it's a test, but there's no var comment... It should probably go to the definition section above.
```
  protected $image;
  protected $non_image;
  protected $file;
```
@argiepiano what do you think? Add it or not?

Reply:
Thanks, @indigoxela. Added!